### PR TITLE
Add issue and pull request templates

### PR DESCRIPTION
**Pull request by andrew**
From https://github.com/librariesio/documentation

cc @BenJam

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Thanks taking the time to contribute. This template should help guide you through the process of creating a pull request for review. Please erase any part of this template that is not relevant to your pull request:
+
+
+- [ ] Have you followed the guidelines for [contributors](http://docs.libraries.io/contributing)?
```
Comment (BenJam) [inline .github/pull_request_template.md]: Link needs fixing first. 🤕

Comment (BenJam) [inline .github/pull_request_template.md]: See https://github.com/librariesio/documentation/pull/24

**Comment by BenJam** [comment]
Just updated the template and proposed moving from current contributors to new template. For you to merge if okay.

**Comment by andrew** [comment]
@BenJam I see you removed the contributors.md from this repo, we should just have a minimal one so it still shows up when opening issues: 
screen shot 2017-02-16 at 2 24 34 pm

Also GitHub is starting to judge project "health" by the presence of contributing.md

**Pull request merged**